### ISSUE TITLE
Docs: Proofreading edited JOSS paper

### ISSUE THREAD

**Issue by NMontanaBrown** [DeepRegNet/DeepReg#405]
## Subject of the issue

Paper was too long, so @mathpluscode and @YipengHu edited it to an appropriate length. Content may have changed so we recommend a review before merge into main.

@mathpluscode @YipengHu can you point to the commit and make a PR with all authors once the new document is ready? Thanks 😄

**Comment by NMontanaBrown** [comment]
@mathpluscode @ebonmati @MattClarkson @YipengHu @RemiDelaunay @YipengHu @QianyeYang @acasamitjana @agrimwood @zacbaum @ZheMin-1992 @s-sd

**Comment by YipengHu** [comment]
This is already in Main due to requirement from the reivew: https://github.com/DeepRegNet/DeepReg/blob/main/docs/joss_paper/paper.md

Please leave comments here for now?

**Comment by mathpluscode** [comment]
Please check https://github.com/DeepRegNet/DeepReg/blob/main/docs/joss_paper/paper.md or this PDF
[DeepReg.pdf](https://github.com/DeepRegNet/DeepReg/files/5252399/o4qivbb1mht2as5xxuxw.pdf) for at least

- Name
- Institution
- orcid (using https://raw.githubusercontent.com/DeepRegNet/DeepReg/main/docs/joss_paper/paper.md)
- Acknowledgement

If there's no problem, maybe we can close this issue.

@mathpluscode @ebonmati @MattClarkson @YipengHu @RemiDelaunay @YipengHu @QianyeYang @acasamitjana @agrimwood @zacbaum @ZheMin-1992 @s-sd

**Comment by YipengHu** [comment]
CMIC, Medical Physics, UCL
Martinos Center for Biomedical Imaging, Massachusetts General Hospital and Harvard Medical School
Computer Science and AI Lab (CSAIL), Massachusetts Institute of Technology

**Comment by NMontanaBrown** [comment]
Ok with me.

**Comment by acasamitjana** [comment]
Fine with me, just update Eugenio's affiliation with Yipeng's comment

**Comment by RemiDelaunay** [comment]
It looks good for me too
